Flow.Launcher.Plugin/Result.cs

```diff
@@ -36,6 +36,9 @@ public class Result
         /// </summary>
         public string AutoCompleteText { get; set; }
 
+        /// <summary>
+        /// Image Displayed on the result
```

Comment:
I think it would help to point out what kind of values are expected here.. I assume we only support local file paths (do we support URLs here yet?)

Also, let's make it clear that this is ignored if `Glyph` is set...